portfolio/src/main/java/com/google/sps/servlets/DeletionServlet.java

```diff
@@ -56,12 +59,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     // Initialise a connection to DataStore.
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    BlobstoreService blobstore = BlobstoreServiceFactory.getBlobstoreService();
 
     // Each key has a unique id associated with it.
     // This id can be used to reconstruct the key using .createKey()
     Key key = KeyFactory.createKey("Comment", body.getId());
+    BlobKey blobKey = new BlobKey(body.getImageBlobstoreKey());
```

Comment:
Is it possible for the comment to not have an associated BlobKey, and if so, does it error out here?

Reply:
This was a known bug yes. I had fixed it prior to review but hadn't pushed the changes from my local repo. Fixed now. Sorry about the confusion.